src/DapperQueryBuilder/FluentQueryBuilder/IFromBuilder.cs

```diff
@@ -9,6 +9,16 @@ namespace DapperQueryBuilder
     /// </summary>
     public interface IFromBuilder : ICompleteCommand
     {
+        /// <summary>
+        /// Adds one column to the select clauses
+        /// </summary>
+        ISelectBuilder Select(FormattableString column);
```

Comment:
I understood the `GroupBy()`, but I guess these `Select()` methods aren't supposed to be here.
How could we possibly add a `SELECT` after a `FROM` clause?

Reply:
Sorry, I added these after creating my pull request as they are useful for us in our use case. I didn't realize they'd be included. 

For reference, we have user defined tables and allow them to choose which fields to include. Your library was almost perfect for our use case but we wanted to be able to create a base query and then append the necessary joins, selects, etc based on the fields to be included. It may seem odd to add a select after a from, but based on how the SQL is ultimately constructed by your library doing something like this works out fine.

```
var dataQuery = cn.FluentQueryBuilder()
    .Select($"{list.InternalName:raw}.Id")
    .From($"{list.InternalName:raw}");

foreach (var viewField in view.ViewFields)
{
    viewField.AddToQuery(dataQuery); // Adds additional selects, joins, etc as needed by the field being added
}
```

If you don't want to include the added Select options here, they can be removed. We will just continue to use our own fork :)